samples: openthread: remove data lenght check for ble data (#7603)
nrfconnect/sdk-nrf: samples/openthread/coap_client/README.rst

Multiprotocol extension requirements
====================================

> If you enable the :ref:`coap_client_sample_multi_ext`, make sure you have a phone with the `nRF Toolbox`_ application installed or additional development kit programmed with :ref:`central_uart` sample.

Reviewer: ```suggestion
If you enable the :ref:`coap_client_sample_multi_ext`, make sure you have a phone with the `nRF Toolbox`_ application installed or an additional development kit programmed with :ref:`central_uart` sample.
```

Testing multiprotocol Bluetooth LE extension
--------------------------------------------

> To test the multiprotocol Bluetooth LE extension, you can use nRF Toolbox or :ref:`central_uart` sample.

Reviewer: ```suggestion
To test the multiprotocol Bluetooth LE extension, you can use nRF Toolbox or the :ref:`central_uart` sample.
```

> Below steps consider nRF Toolbox as bluetooth tester.
> Firstly you need to set up nRF Toolbox as follows:

Reviewer: ```suggestion
The steps below assume nRF Toolbox as the Bluetooth tester.

First, you need to set up nRF Toolbox as follows:
```
